resources/views/hr/employees/index.blade.php

```diff
@@ -15,9 +15,9 @@
                     <option {{ $filters['status'] == 'previous' ? "selected=selected" : '' }} value="previous">Previous</option>
                 </select>
             </div>
-            <div class="d-flex align-items-center ml-35">
-                    <input type="text" name="employeename" class="form-control" id="name" placeholder="Enter the Employee" value="{{ request()->get('employeename') }}">
-                    <button class="btn btn-info ml-2 text-white">Search</button>
+            <div class="d-flex align-items-center ml-25">
+                    <input type="text" name="employeename" class="form-control" id="name" placeholder="Enter the Employee" value="{{ request()->get('employeename') }}" button class="pr-3.0">
```

Comment:
Please let me know, how much size of the `pr-3.0`. and also the what is means of the `button` in this input field.
```suggestion
                    <input type="text" name="employeename" class="form-control pr-3.0" id="name" placeholder="Enter the Employee" value="{{ request()->get('employeename') }}" button class="">
```

Reply:
This **button**  is added by mistake , and I have corrected it .